packages/networks/src/networks/vNaga/shared/managers/api-manager/executeJs/executeJs.ts

```diff
@@ -363,6 +377,7 @@ export const handleResponse = async (
 
   const hasSignedData = Object.keys(mostCommonResponse.signedData).length > 0;
   const hasClaimData = Object.keys(mostCommonResponse.claimData).length > 0;
+  const paymentDetail = mostCommonResponse.paymentDetail;
```

Comment:
this is an interesting detail - the paymentDetail will likely be different from every node.  technically we should expose the value from all the nodes, right?  like a map of node ip address -> paymentDetail?

or we could sum them up instead of returning a map, so the user just sees their "total costs".  i think we should do that?  and then, make sure all the individual responses are logged via the debug logger, so they can see the individual nodes paymentDetail if they're debugging stuff, but by default, they just see the total cost of the requests.

what do you think?  summing + debug logs?